packages/components/psammead-useful-links/src/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import { C_EBON } from '@bbc/psammead-styles/colours';
+import { grid } from '@bbc/psammead-styles/detection';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { string, arrayOf, shape } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  GEL_SPACING, // 8 px
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+
+const UsefulLinkWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
```

Comment:
This will need to be confirmed by UX, and be consistent with the margin we have between the section labels and the first promo in the rest of the sections of the page.

Reply:
In fact, I think we shouldn't be adding a wrapper with padding/margin to this component. The spacing should be handled in Simorgh like we do in the [FrontPageSection](https://github.com/bbc/simorgh/blob/latest/src/app/containers/FrontPageSection/index.jsx#L129) container. Thoughts @sadickisaac?

Reply:
@DenisHdz You are right spacing should be handled in simorgh that way we make our components more reusable. Think what happens if someone uses this component and is not happy with the padding and would like to increase/decrease it. Would he/she need to first change the component and then pull it back into his/her project.

Reply:
Sounds good, I removed all the outer paddings/margins